docs/www/configuration.md

```diff
@@ -622,6 +622,19 @@ rpk:
   # Default: ''
   coredump_dir: "/var/lib/redpanda/coredump"
 
+  # Creates a "ballast" file that can be removed in the odd case that a redpanda node
+  # runs out of space, so that the node can be restarted and troubleshot.
+  # Default: false
+  tune_ballast_file: false
```

Comment:
I think it may be worth it to have an inner struct,
```yaml
ballast_file:
    path: /var/lib/...
    size: 1GiB
```
where the presence of the struct determines whether to create a ballast file or not (i.e., the presence replaces `tune_ballast_file`). This would have the benefit of grouping all ballast file options.

Reply:
I _think_ we can do this. There's some machinery in place that I need to check doesn't break if we change the approach for this tuner, but I can't think of anything that would cause it to. Will get back to you today.

Reply:
As discussed over slack, we'll just leave it as is, since the defaults are good enough and `tune_ballast_file: true` will be ok for most cases.